### PR TITLE
create robstalnaker plano recomendation file

### PR DESCRIPTION
**Pull request by robstalnaker**
- this is a bullet point. 
-[] this is a task list item

### CODE REVIEW

File: robstalnaker-LiveFromPlanoTexas.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Plano Texas
+
+### Places to Eat
+- my moms house
```
Comment (mohitspawar) [inline robstalnaker-LiveFromPlanoTexas.md:4]: Is there 🎵 and 🐷 BBQ available ?

Comment (robstalnaker) [reply robstalnaker-LiveFromPlanoTexas.md:4]: yes

**Review by mohitspawar — APPROVED** [review]
Go ahead and merge 👍

**Comment by robstalnaker** [comment]
finally !

**Pull request merged**